packages/gestalt/src/Dropdown.js

```diff
@@ -23,13 +23,16 @@ function getChildrenOptions(childrenArray) {
   return childrenArray.reduce((accumulatedChildren, currentChild) => {
     const {
       props: { children: currentItemChildren },
+      type,
       type: { displayName },
     } = currentChild;
 
-    if (currentItemChildren && displayName === 'Dropdown.Section') {
+    if ((currentItemChildren && displayName === 'Dropdown.Section') || typeof type === 'symbol') {
```

Comment:
This isn't really readable. Consider `child.type === Fragment`, or even better creating a helper function `isFragment` that encapsulates that logic

Reply:
DONE. Thanks!